packages/@okta/vuepress-site/docs/guides/sign-into-spa/use-the-access-token/index.md

```diff
@@ -14,4 +14,11 @@ On your back-end (the API), make sure that you check for valid tokens in incomin
 
 <StackSelector snippet="getaccesstoken"/>
 
+<br/>
+
+In order to enable access token renewal it is required to obtain a refersh token. See [Get a refresh token with the code flow](/docs/guides/refresh-tokens/get-refresh-token/#get-a-refresh-token-with-the-code-flow).
```

Comment:
```suggestion
In order to enable access token renewal it is required to obtain a refresh token. See [Get a refresh token with the code flow](/docs/guides/refresh-tokens/get-refresh-token/#get-a-refresh-token-with-the-code-flow).
```

Reply:
thanks - this got included into one of the suggested changes below